### ISSUE TITLE
Broken responsive web design pattern  with  Angular  +  bootstrap 4 + flex or a new feature

### ISSUE THREAD

**Issue by mi9rom** [ag-grid/ag-grid#3009]
Hello, 
I need to know,  is it a bug and will be fixed or it is a new feature?
I'm a software architect in a big project and I have to make decisions regarding using this component (especially switching to the enterprise version).

**Providing a Reproducible Scenario**
Simply compare AG v19.1.4 
https://angular-ag-grid19-vbar-flex-no-problem.stackblitz.io
with AG v20.1.0 
https://angular-ag-grid20-vbar-flex-real-problem.stackblitz.io

I think all other info You need, You can find here:
https://stackblitz.com/edit/angular-ag-grid20-vbar-flex-real-problem
https://stackblitz.com/edit/angular-ag-grid19-vbar-flex-no-problem

Angular  7 + ag-grid  20.1.0  + bootstrap 4  flex. 

Best

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by jiangyh1024** [comment]
I think this is the same issue as #2873. you can check it out.

**Comment by mi9rom** [comment]
Thank You @jiangyh1024 
Your workaround looks promising.  :)
Best

**Comment by jiangyh1024** [comment]
@mi9rom 
```
  .ag-root-wrapper-body.ag-layout-normal {
    flex-grow: 1;
    height: 0px;
    min-height: 0px;
  }
```
should be 
```
  .ag-root-wrapper-body.ag-layout-normal {
    flex: 1 1 auto;
    height: 0px;
    min-height: 0px;
  }
```
or it will have problem using firefox, found lately

**Comment by makinggoodsoftware** [comment]
Hi,

This issue has been fixed in version 20.2.0
